casts/rows/src/UIAccountRow.tsx

```diff
@@ -25,7 +25,7 @@ export type UIAccountRowProps = {
 
 const zeroBigNumber = new BigNumber(0);
 
-export function UIAccountRow({
+export const UIAccountRow = React.memo(function UIAccountRow({
```

Comment:
👍 Thanks! I believe we'll need to inspect all our code to apply React.memo whenever it's appropriate... Or just wait for React Forget 😄